javascript/ql/src/semmle/javascript/security/dataflow/Xss.qll

```diff
@@ -329,7 +329,7 @@ module DomBasedXss {
    * A write to the `template` option of a Vue instance, viewed as an XSS sink.
    */
   class VueTemplateSink extends DomBasedXss::Sink {
-    VueTemplateSink() { this = any(Vue::Instance i).getTemplate() }
+    VueTemplateSink() { this = any(Vue::Instance i).getOption("template") }
```

Comment:
I suppose this is for performance: should we have a comment here? It seems too easy to think that `.getOption("template")` should be refactored to `.getTemplate()`

Reply:
It's because `getTemplate` does `getALocalSource()`, which is generally not what you want when adding a sink. Added a comment to explain.